### PR TITLE
Contributors updates for Chapel 1.29

### PR DESCRIPTION
**Pull request by bradcray**

### CODE REVIEW

**Comment by bradcray** [comment]
@chapel-lang/core-contributors : If you reviewed and merged PRs from external contributors, please be sure that they're listed here.  The approach I take for populating this is lossy (possibly due to peoples' privacy settings).

@vwang0101: If you would check that I have spelled and alphabetized your name correctly, I would appreciate it—thanks for your contributions!

File: CONTRIBUTORS.md [hunk 1]
```diff
@@ -10,7 +10,6 @@ Contributors to the Chapel 1.28.0 release
 * Paul Cassella, [HPE]
```
[the remark this answers is not in the document]
Comment (bradcray) [reply CONTRIBUTORS.md:10]: Yep, thanks for the catch... going too fast...

File: CONTRIBUTORS.md [hunk 2]
```diff
@@ -20,18 +19,15 @@ Contributors to the Chapel 1.28.0 release
...
 * Vassily Litvinov, [HPE]
 * David Longnecker, [HPE]
 * Ben McDonald, [HPE] (former intern from [Gonzaga University])
-* Brandon Neth, [HPE] intern from [University of Arizona]
-* Nikki Rad, [HPE] intern from [Texas A&M University]
 * Ahmad Rezaii, [HPE]
 * Anna Rift, [HPE]
-* Thomas Rolinger, individual contributor
 * Elliot Ronaghan, [HPE]
 * Andy Stone, [HPE] (former [Cray Inc.] intern from [Colorado State University])
```
Comment (mstrout) [inline CONTRIBUTORS.md:28]: @bradcray , this seems to imply that Andy (and Ben McD above) made the contributions while they were interns?  It struck me as odd.  I am not sure why we are mentioning they used to be an intern here?

Comment (bradcray) [reply CONTRIBUTORS.md:28]: The intention of the wording is to nod to their past contributions and affiliations as interns on top of their present ones as HPE developers.  For devs in this category, I've given them the option to nod to their roots or shed them, where different people have gone different ways (Engin retired his in the past year or so I think, Andy and BenMcD preferred keeping theirs last I asked).

Comment (bradcray) [reply CONTRIBUTORS.md:28]: (closing this tab at the end of the day:  feel free to offer a suggested wording that would make it clearer.  I suspect the reason I used this terse wording was to stay as close to 80 columns as possible).

**Pull request merged**